src/Core/Resources/Resources.hpp

```diff
@@ -0,0 +1,23 @@
+#ifndef RADIUMENGINE_RESOURCES_HPP
+#define RADIUMENGINE_RESOURCES_HPP
+#include <Core/RaCore.hpp>
+
+namespace Ra {
+namespace Core {
+
+/** Resources management for Radium.
+ *  This namespace contains functions, classes and utilities for resource management.
+ *  Resources are :
+ *    - configuration file (ui, keymapping, ...)
+ *    - default shaders
+ *    - ...
+ *
+ *    @todo define and implement a portable interface for resource management.
+ */
```

Comment:
Shouldn't there be one in `Engine` (for shaders) and one in `GuiBase` (for ui images), rather than one in Core which do not define/use resources?

Reply:
It is in core because it is used by both guibase, Engine (and why not, by IO etc ...)
But this 1st version is definitively not the one that will be used in V2. First step toward (and headhache removal when using application and plugins from outside the Radium Bundle)